src/beman/exemplar/CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-add_library(beman.exemplar STATIC)
+add_library(beman.exemplar INTERFACE)
```

Comment:
If we change this we should enable VERIFY_INTERFACE_HEADER_SETS at the same time. Compiling the header is the purpose of having the cpp file. 

Also, header only is more of an accident than a plan, and since this also serves as a template for further work I'm not sure if INTERFACE should be the default?

Reply:
see too https://github.com/beman-project/exemplar/issues/59